docs/pages/enroll-resources/kubernetes-access/health-checks.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Teleport Kubernetes Health Checks
+sidebar_label: Health Checks
+description: How to configure Teleport Kubernetes health checks and view health.
+tags:
+- conceptual
+- zero-trust
+- infrastructure-identity
+---
+
+This documentation provides an overview of health checks to Kubernetes clusters with Teleport. Teleport Kubernetes Services periodically check the connectivity and permissions of enrolled Kubernetes clusters, and is available in Teleport version `18.3.0` and later.
```

Comment:
```suggestion
This documentation provides an overview of health checks to Kubernetes clusters with Teleport. Teleport Kubernetes Service instances periodically check the connectivity and permissions of enrolled Kubernetes clusters, and is available in Teleport version `18.3.0` and later.
```

Reply:
probably should just `sed 's#Kubernetes Services#Kubernetes Service instances#g`